apps/services/sessions/infra/sessions.ts

```diff
@@ -16,6 +16,7 @@ const workerPostgresInfo = {
   username: 'services_sessions',
   name: dbName,
   passwordSecret: '/k8s/services-sessions/DB_PASSWORD',
+  extensions: ['uuid-ossp'],
```

Comment:
Nice!
We should also add this to `apps/services/auth/ids-api/infra`

Reply:
> Nice! We should also add this to `apps/services/auth/ids-api/infra`

done